cloud/azure/database/redis/plugin.pm

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2021 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package cloud::azure::database::redis::plugin;
+
+use strict;
+use warnings;
+use base qw(centreon::plugins::script_custom);
+
+sub new {
+    my ($class, %options) = @_;
+    my $self = $class->SUPER::new( package => __PACKAGE__, %options );
+    bless $self, $class;
+
+    $self->{version} = '0.1';
+    $self->{modes} = {
+        'discovery' => 'cloud::azure::database::redis::mode::discovery',
+        'cache-hits'   => 'cloud::azure::database::redis::mode::cachehits',
+        'cache-latency'   => 'cloud::azure::database::redis::mode::cachelatency',
+        'cache-miss-rate'   => 'cloud::azure::database::redis::mode::cachemissrate',
+        'connected-clients'   => 'cloud::azure::database::redis::mode::connectedclients',
+        'server-load'   => 'cloud::azure::database::redis::mode::serverload',
+        'operations'   => 'cloud::azure::database::redis::mode::operations',
+        'cpu'   => 'cloud::azure::database::redis::mode::cpu',
+        'memory'   => 'cloud::azure::database::redis::mode::memory',
+        'errors'   => 'cloud::azure::database::redis::mode::errors',
+        'cache-throughput'   => 'cloud::azure::database::redis::mode::cachethroughput'
```

Comment:
Sort & align please
Also please rename the modes as discussed in the main conversation ;)
Thanks!